briskedu/brisk-learning
# Fix segment logging

Summary:
Fixing logging by adding back in host.
In next PR I'll add in testing to make sure
this is caught statically

Test Plan:
1. Confirm logging works from front-end
2. Confirm logged out logging works
